### PR TITLE
Bump decidim from v0.28.1 to v0.29.0.rc3

### PR DESCRIPTION
**Pull request by andreslucena**
This PR updates Decidim following the releases notes from https://github.com/decidim/decidim/releases/tag/v0.29.0.rc1 

The only difference is that I've changed the rc1 to rc2 as the rc1 had a bug with a migration and the upgrade process didn't work.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,21 +2,21 @@
   "name": "metadecidim",
   "private": true,
   "dependencies": {
-    "@decidim/browserslist-config": "^0.28.1",
-    "@decidim/core": "^0.28.1",
+    "@decidim/browserslist-config": "~0.29.0-rc2",
+    "@decidim/core": "~0.29.0-rc2",
     "@decidim/elections": "^0.28.1",
```
Comment (fblupi) [inline package.json:7]: Why didn't you update the `@decidim/elections` package?

**Review by fblupi — COMMENTED** [review]
Just a small question. I tested it locally and runs as expected!

File: package.json (hunk 1 above)
Comment (andreslucena) [reply package.json]: Nice catch, I missed this one! 

We (@decidim/maintainers) actually dropped official support of decidim-elections in this version. In the next few months @decidim/product will do a blog post explaining in detail what happened. Just as a summary it was a development that isn't maintained anymore, so it's safe to drop it.

Comment (andreslucena) [reply package.json]: I removed it on [a8d9860](https://github.com/decidim/metadecidim/pull/129/commits/a8d986070e4c3cc36ef4072a9d7572e14a14ff6f)

**Review by fblupi — APPROVED** [review]

File: db/migrate/20240717080156_add_withdrawn_at_field_to_proposals.decidim_proposals.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    STATES = { not_answered: 0, evaluating: 10, accepted: 20, rejected: -10, withdrawn: -20 }.freeze
+    enum state: STATES, _default: "not_answered"
+  end
+
+  def up
+    add_column :decidim_proposals_proposals, :withdrawn_at, :datetime
+
+    CustomProposal.withdrawn.find_each do |proposal|
+      proposal.withdrawn_at = proposal.updated_at
+      proposal.save!
```
Comment (alecslupu) [inline db/migrate/20240717080156_add_withdrawn_at_field_to_proposals.decidim_proposals.rb:16]:
```suggestion
      proposal.state = :not_answered
      proposal.save!
```

**Comment by andreslucena** [comment]
@fblupi 

we implemented a `7z` CLI wrapper as a workaround with the lack of maintenance of the `seven_zip_ruby` gem. You should add this buildpack to the heroku configuration before deploying: https://elements.heroku.com/buildpacks/opendoor-labs/heroku-buildpack-p7zip (We will add these instructions in the Releases Notes for the v0.29.0)

**Pull request merged**